tests/opentelemetry/test_trace.py

```diff
@@ -161,12 +161,8 @@ def test_otel_start_current_span_without_default_args(oteltracer):
 )
 @pytest.mark.snapshot(ignores=["metrics.net.peer.port", "meta.traceparent", "meta.flask.version"])
 def test_distributed_trace_with_flask_app(flask_client, oteltracer):  # noqa:F811
-    with oteltracer.start_as_current_span("test-otel-distributed-trace") as otel_span:
-        headers = {
-            "traceparent": otel_span._ddspan.context._traceparent,
-            "tracestate": otel_span._ddspan.context._tracestate,
-        }
-        resp = flask_client.get("/otel", headers=headers)
```

Comment:
Why did you make this change?

Reply:
The way it was originally written we generated headers, and then sent them through manually. However, if you just rely on the inject method, it generates these headers anyways, and then in addition runs sampling. 

The alternative is that we could manually run sampling and then manually inject that header, but it made more sense to me to test more of how this would actually be used in the wild.